src/Evolution/Executables/NewtonianEuler/EvolveNewtonianEuler.hpp

```diff
@@ -78,6 +80,34 @@ namespace Parallel {
 template <typename Metavariables>
 class CProxy_ConstGlobalCache;
 }  // namespace Parallel
+
+namespace detail {
+template <typename Metavariables,
+          bool = not tmpl::list_contains_v<
+              tmpl::list<typename Metavariables::source_term_type>,
+              NewtonianEuler::Sources::NoSource>>
```

Comment:
`cpp17::is_same_v<typename Metavariables::source_term_type, NewtonianEuler::Sources::NoSource>`, and use `tmpl::conditional_t`.  Again below.  (Same as is done with the local time stepping stuff.)